### PR TITLE
Allow symfon/twig-bridge 3

### PR DESCRIPTION
**Pull request by mvanbaak**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "php": ">=5.3.0",
         "silex/silex": ">=1.0,<2.0",
         "twig/twig": ">=1.8,<2.0-dev",
-        "symfony/twig-bridge": "~2.3"
+        "symfony/twig-bridge": "~2.3|^3.0"
```
Comment (nymo) [inline composer.json]: Looking at the [docs from composer](https://getcomposer.org/doc/articles/versions.md#range) this should be a double pipe or?

Comment (mvanbaak) [inline composer.json]: Everywhere I see this it's a single pipe.
Check for example silexphp/silex

Comment (mvanbaak) [inline composer.json]: https://github.com/silexphp/Silex-WebProfiler/blob/master/composer.json

Comment (nymo) [inline composer.json]: Mhm I see maybe it's a typo in the documentation or both versions are possible.

**Pull request merged**